### DISCUSSION TITLE
Draw contour for detection phase

### DISCUSSION THREAD

**Discussion by dvivarelli** [mindee/doctr#706] [Q&A]
Hello,
I would like to draw the result contour of DBNet prediction.
Does an utility for this porpouse already exists?

**Comment by charlesmindee** [comment]
Hi @dvivarelli,

Of course you can do that, you have functions in`utils/visualization.py`, and after calling an ocr_predictor on an image you can just do the following:
```
result = ocr_predictor(doc)
result.show(image) 
```
And it will display the boxes on the image

I hope I answered your question!

have a nice day

**Reply by charlesmindee** [reply]
If you are talking about raw DB predictions, you can use the streamlit demo to see the heatmap or you can directly retrieve the heatmap and the binary map from the `DetectionPostProcessor` in the `core.py` file of the detection module (in the call). We don't provide tools for now to easily plot the output of the detection model.
